pkgs/applications/networking/feedreaders/rsstail/default.nix

```diff
@@ -21,15 +20,16 @@ stdenv.mkDerivation {
   makeFlags = [ "prefix=$(out)" ];
   enableParallelBuilding = true;
 
-  doCheck = true;
+  # just runs cppcheck linter
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:**

```suggestion
  # there no test, only Zuul
```

Reading this comment without the context of this PR is crypt, because what we are saying is that. "We won't run test, because there is no test, only linter"